device/src/mac/mod.rs

```diff
@@ -125,6 +133,9 @@ impl Mac {
         let mut otaa = otaa::Otaa::new(credentials);
         let dev_nonce = otaa.prepare_buffer::<C, RNG, N>(rng, buf);
         self.state = State::Otaa(otaa);
+        let mut tx_config =
+            self.region.create_tx_config(rng, self.configuration.data_rate, &Frame::Join);
+        tx_config.adjust_power(self.board_eirp.max_power, self.board_eirp.antenna_gain);
         (self.region.create_tx_config(rng, self.configuration.data_rate, &Frame::Join), dev_nonce)
```

Comment:
This looks odd here. looks like `tx_config` is updated with board_eirp data but then not used.

Reply:
Good catch again! Thanks